client/src/charts/wrapped_nivo/WrappedNivoPie.js

```diff
@@ -182,6 +180,11 @@ WrappedNivoPie.defaultProps = {
     bottom: 60,
     left: 50,
   },
+  theme: {
```

Comment:
Inconsistency from Nivo: This `boxShadow` exists for Nivo bar and circle-pack but not for line and pie. So can't put this all together into nivo_common tooltip factory. I'll try making an issue on Nivo but they'll probably make me start a mini project to replicate it🤮